engine/syncer.go

```diff
@@ -18,8 +18,9 @@ const (
 	SyncItemOrderbook
 	SyncItemTrade
 
-	DefaultSyncerWorkers = 15
-	DefaultSyncerTimeout = time.Second * 15
+	DefaultSyncerWorkers          = 15
+	DefaultSyncerTimeoutREST      = time.Second * 15
+	DefaultSyncerTimeoutWebsocket = time.Minute
```

Comment:
As discussed, the issue I've experienced with this is with an Exchange like Binance. Where an asset type which is unsupported by the websocket is delayed by a minute in order to switch to REST. The initial sync cannot complete until thie minute expires and sync continues. My short term solution is to reduce this timer or have an ability to identify if an asset type cannot be synced via websocket even if websocket is true. Anything larger requires its own PR

Reply:
Have added a quick fix for this but I do get this warning when spot is not enabled, which we can probably address later: [WARN] |WEBSOCKET| 09/04/2021 10:26:11 |Binance websocket has been disconnected. Reason: read tcp 192.168.1.109:51983->52.68.174.232:9443: use of closed network connection
I am assuming there is nothing to subscribe so it opens a connection and then returns.

Reply:
I can confirm that its the reason for the error. I don't mind this so much, the user has been warned that the asset is unsupported. I don't think we need to forcefully disable the websocket on them, the user can still make modifications via GRPC or they can choose to disable the websocket themselves/enable a supported asset.